src/domains/transaction/components/BusinessRegistrationForm/BusinessRegistrationForm.tsx

```diff
@@ -321,7 +321,7 @@ export const BusinessRegistrationForm: RegistrationForm = {
 					// @TODO: let the user choose what sub-type they wish to use.
 					subType: Enums.EntitySubType.None,
 					// @TODO: use the name that the user entered. Has to be valid like a delegate username.
-					name: undefined,
+					name: ipfsData.meta.displayName,
```

Comment:
The name and display name are different values as stated by the comment. Something like `slugify(ipfsData.meta.displayName)` would work or letting the user enter the display name.

Reply:
What about `constantCase` from `@arkecosystem/utils`?

Reply:
That would force the name to always be uppercase and not work as intended.

https://github.com/ArkEcosystem/core/blob/develop/packages/core-magistrate-crypto/src/transactions/utils/entity-schemas.ts#L19
https://github.com/ArkEcosystem/core/blob/develop/packages/core-magistrate-crypto/src/transactions/utils/entity-schemas.ts#L5-L10